packages/edit-post/src/components/layout/style.scss

```diff
@@ -99,3 +99,15 @@
 .edit-post-layout .entities-saved-states__panel-header {
 	height: $header-height + $border-width;
 }
+
+.edit-post-layout.has-fixed-toolbar {
+	// making the header be lower than the content
+	// so the fixed toolbar can be positioned on top of it
+	// but only on desktop
+	@include break-medium() {
+		.interface-interface-skeleton__header:not(:focus-within) {
+			z-index: 19;
```

Comment:
Does it make sense to use variables for these z-index values so that we can see how they relate to each other?

Reply:
Maybe ... but how would the variable name explain it better than the comment? It's a weird situation.

Reply:
We have a file for this. I believe when I was looking for the z-index values the line I looked for was [this one](https://github.com/WordPress/gutenberg/blob/trunk/packages/base-styles/_z-index.scss#L32). It would make sense to store them in context.

Reply:
How? Swap the values?

Reply:
Like this:

`z-index: z-index(".interface-interface-skeleton__header:not(:focus-within)");`

And add the relevant comments to the z-index file, having the new selector next to the ones that are relevant to it.

Reply:
Ok I'll do that. Tho to me it only seems I add more hoops to jump through for the future developer reading this hack :)